cypress/integration/hovercard-controller.spec.js

```diff
@@ -0,0 +1,150 @@
+context('HovercardController', () => {
+  beforeEach(() => {
+    cy.visit('/docs/hovercard-testing.html')
+  })
+
+  context('anchor is 0 to left of the viewport', () => {
+    it('hovercard is at top of the anchor', () => {
+      cy.get('#first').as('anchor')
+      cy.get('@anchor').trigger('mouseenter')
+      cy.get('div.hovercard:not(.is-hidden)').as('hovercard')
+      let anchorTop, widthHovercard, leftHovercard, leftSvg
+
+      cy.get('@anchor').then($anchor => {
+        anchorTop = $anchor[0].getBoundingClientRect().top
+      })
+
+      cy.get('@hovercard').then($hovercard => {
+        expect(
+          $hovercard[0].getBoundingClientRect().top +
+            $hovercard[0].getBoundingClientRect().height
+        ).to.be.lessThan(anchorTop)
+        widthHovercard = $hovercard[0].getBoundingClientRect().width
+        leftHovercard = $hovercard[0].getBoundingClientRect().left
+      })
+
+      cy.get('div.hovercard:not(.is-hidden) svg').then($svg => {
+        leftSvg = $svg[0].getBoundingClientRect().left
+        expect(leftSvg - leftHovercard).to.be.lessThan(widthHovercard / 2)
+      })
+    })
+
+    it('hovercard is at botttom of the anchor', () => {
+      cy.get('#second').as('anchor')
```

Comment:
```suggestion
      cy.get('#topLeft/topCenter/topRight').as('anchor')
```
Para identificar más facil la prueba

Reply:
buena recomendacion!